MOBT-3-Patterns/MVVMWithoutC/Login/LoginViewModel.swift

```diff
@@ -0,0 +1,91 @@
+//
+//  LoginViewModel.swift
+//  MVVMWithoutC
+//
+//  Created by Mykyta Babanin on 01.07.2021.
+//
+
+import Foundation
+
+
+protocol LoginViewModelProtocol {
+    typealias authenticationLoginCallBack = (_ status: Bool) -> Void
```

Comment:
Status keyword assumes some sort of enum or status object. Let's use that or rename the argument to `success` keyword